lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -728,9 +729,10 @@ void ProxyRenderDelegate::_Execute(const MHWRender::MFrameContext& frameContext)
         }
 #endif
     } else {
-        if (_selectionChanged) {
+        if (_selectionChanged || _selectionModeChanged) {
```

Comment:
The _selectionModeChanged work is deliberately not protected by the preprocessor macro. I'm planning to use it in my next change as well and that one doesn't have any Maya version dependencies.